### PR TITLE
Raise minimum PHP requirements to ^8.0

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️
| Fixed issues  | -

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -20,12 +21,11 @@ jobs:
           - windows-latest
 
         php:
-          - "7.4"
           - "8.0"
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v2.3.4
```
Comment (samdark) [inline .github/workflows/build.yml:28]: It was alright.

File: .github/workflows/static.yml [hunk 2]
```diff
@@ -16,11 +16,11 @@ jobs:
           - ubuntu-latest
 
         php:
-          - "7.4"
+          - "8.0"
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v2.3.4
```
Comment (samdark) [inline .github/workflows/static.yml:23]: It was alright.

File: .github/workflows/build.yml (hunk 1 above)
Comment (vjik) [reply .github/workflows/build.yml]: Done.

File: .github/workflows/static.yml (hunk 2 above)
Comment (vjik) [reply .github/workflows/static.yml]: Done.

**Review by samdark — APPROVED** [review]

**Pull request merged**